doc/source/user_guide/documentation/icepyx.rst

```diff
@@ -23,4 +23,5 @@ Diagrams are updated automatically after a pull request (PR) is approved and bef
    query
    read
    quest
+   variables
```

Comment:
Note to remove variables content from `components.rst`

Reply:
good catch!